### ISSUE TITLE
Check map load performance

### ISSUE THREAD

**Issue by andygup** [jwasilgeo/ionic-esri-map#7]
Time to first tile load seems slow from the time the map tab is opened. Needs investigation.

**Comment by justingreco** [comment]
Just submitted an Ionic app using the JS 4.0 API to through iTunes Connect.  It was just rejected due to the "loading wheel spinning indefinitely".  Wonder if it could be related to this issue.  I have noticed that it can take up to 10 to 12 seconds for the map to fully load.   Have you found any resolution for this?

**Comment by andygup** [comment]
@justingreco Nope, haven't looked at the initial load performance yet. Also just a gentle note that all the work in this repo is 100% experimental. 

We are still looking into **JS API v3.16-only** + Ionic + Angular general memory management issues. Details available under [Issue 9](https://github.com/jwasilgeo/ionic-esri-map/issues/9).

And another important note: for web mobile, it is still recommended to stick with JS API 3.x  since it has been well tested in the cross-browser, web mobile environment. Among a number of things, JS API 4.x API won't have complete gesture support until at least v4.1. Full details on this will be forth coming next month at the User Conference.

If you need more info, or want to discuss further offline then I'd be happy to set up a time to chat?

**Comment by andygup** [comment]
@justingreco any luck figuring it out?

**Comment by justingreco** [comment]
@andygup ended up switching to Leaflet and esri-leaflet and the memory is much lower than using the JS API.  May end up testing the JS API with Ionic 2 to see if the memory issue is still present.

**Comment by andygup** [comment]
@justingreco Nice, good call.
